diagnostic.md

```diff
@@ -45,7 +59,18 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+.big{
+
+}
+#contentPane {
+
+}
+.important .red {
 
+}
+div, p {
```

Comment:
This is a way of selecting multiple elements, ```<div>``` and ```<p>```.  ```div > p``` selects ```<p>```s that have a ```<div>``` as a parent.